pkgs/tools/system/fancy-motd/default.nix

```diff
@@ -0,0 +1,46 @@
+{ stdenv, lib, fetchFromGitHub, bc, curl, figlet, fortune, gawk, iproute2, procps }:
+
+stdenv.mkDerivation rec {
+  pname = "fancy-motd";
+  version = "20211305";
```

Comment:
looks like you corrected this in the commit message but not here

YYYY-MM-DD formatting (ISO8601) is preferred
and since this isn't an upstream tagged release,
someone will demand you add `unstable-` in front of this

Reply:
Whoops we submitted this at the same time, and I requested this ;)